client/src/screens/Budget/Budget.jsx

```diff
@@ -1,14 +1,92 @@
 import React from 'react';
+import MaterialTable from 'material-table';
+
 import './Budget.css';
 
 const Budget = () => {
+  const [state, setState] = React.useState({
```

Comment:
don't call it state, call it on the data itself, to call it state and setState is considered a very bad practice